less/pages/encrypt/hybrid-test.less

```diff
@@ -0,0 +1,7 @@
+.encrypt.v3.remove-fields.video4 .page .videoSection .encrypt-signup .signupForm {
```

Comment:
I don't believe there are any other signup forms on that view, so it should be fine.